src/passes/pass.cpp

```diff
@@ -138,19 +138,22 @@ void PassRunner::addDefaultGlobalOptimizationPasses() {
 }
 
 void PassRunner::run() {
-  if (options.debug) {
+  static int passDebug = getenv("BINARYEN_PASS_DEBUG") ? atoi(getenv("BINARYEN_PASS_DEBUG")) : 0;
```

Comment:
That's confusing how that works. It's also unclear that `passDebug` should be consistent across calls to the function.
I think `static const int` better expresses how this is used, and ensures that nobody writes to it after initialization.